project03.py

```diff
@@ -1,7 +1,9 @@
 from prettytable import PrettyTable
 from datetime import date
+import datetime
+import time
 
-GEDCOM_FILE = 'cs555project01.ged'
+GEDCOM_FILE = 'cs555project03.ged'
```

Comment:
We need files as input, I'll handle that during this sprint

Reply:
Yeah he never specified how he wants input